NuRadioMC/simulation/simulation.py

```diff
@@ -465,6 +520,19 @@ def run(self):
 
         t_total = time.time() - t_start
         outputTime = time.time() - t5
+        
```

Comment:
Should we move this to a utility?  This function is in principle handy for all sorts of timing.

Reply:
I mean the code below, sorry if being unclear.